src/types/util/Option.ts

```diff
@@ -1 +1,2 @@
-export type Option<T> = T | null;
+// Not to be confused with the polkadot-js `Codec` extending `Option` type.
+export type IOption<T> = T | null;
```

Comment:
So what is the TS convention for when to use the `I` prefix?

Reply:
I thinks its more just a me convention that I have implicitly picked up from reading other `TS` code - `I` is really for interface, but I use it both for interface or types to avoid name collisions with classes / functions etc.

Reply:
So in typescript (this is quite the debate apparently), "I" should refer to interface's only, and when its not there it will refer to an `impl`. Even though this example doesn't fit those parameters the use of 'I' here is to make sure there is a distinguishing difference as stated in the comment.